packages/backend/src/open_payments/payment/outgoing/model.ts

```diff
@@ -168,6 +183,29 @@ export class OutgoingPayment
       updatedAt: json.updatedAt
     }
   }
+
+  public async getPaymentPointer(): Promise<PaymentPointer> {
+    return this.paymentPointer ?? (await this.$relatedQuery('paymentPointer'))
+  }
+
+  public async toOpenPaymentsType(): Promise<OpenPaymentsOutgoingPayment> {
+    const paymentPointer = await this.getPaymentPointer()
+
+    return {
+      id: await this.getUrl(paymentPointer),
+      paymentPointer: paymentPointer.url,
+      quoteId: (await this.quote?.getUrl()) ?? undefined,
```

Comment:
this is optional in the Open Payments API -> should we keep it?

Reply:
:thinking: I wonder if we should make it required in the spec

Reply:
If you can only create an Outgoing Payment with a required `quoteId`, it would make sense that this would be required here